authenticating-users/main.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2019 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+from flask import Flask
+app = Flask(__name__)
+
+CERTS = None
+AUDIENCE = None
+
+
+def certs():
+    import requests
```

Comment:
Why not import at top of file?  Does our style guide prefer importing at top of file?

Reply:
We have received many user questions about imports when they aren't in the individual snippets (which this will eventually be).

Reply:
What does our current rubric prescribe?

Reply:
It says to show imports, but not specifically where. However, it also says "Shows imports within region tags" which this approach will do once the docs reference region tags instead of containing a static copy. So this seems more in line with the rubric.